antora/package-lock.json

```diff
@@ -10,12 +10,15 @@
       "license": "Apache-2.0",
       "dependencies": {
         "@antora/collector-extension": "^1.0.0-alpha.3",
-        "@antora/lunr-extension": "^1.0.0-alpha.8"
+        "@antora/lunr-extension": "^1.0.0-alpha.8",
+        "@iarna/toml": "^2.2.5",
+        "fs": "^0.0.1-security",
+        "toml": "^3.0.0"
```

Comment:
This could go to devDependencies, it was already pulled in by another dependency transitively it seems